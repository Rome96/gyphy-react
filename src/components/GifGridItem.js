import React from 'react'
import PropTypes from 'prop-types'

const GifGridItem = props => {
  const { urlImg, title } = props;
  return (
    <div>
      <h4>{title}</h4>
      <img src={urlImg} alt={title} />
    </div>
  );
};

GifGridItem.propTypes = {
  urlImg: PropTypes.string.isRequired
};

export default GifGridItem
