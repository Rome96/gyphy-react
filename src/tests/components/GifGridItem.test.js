import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import GifGridItem from '../../components/GifGridItem';

describe("Testing component <GifGridItem/>", () => {
  test("return component GifGridItem/>", () => {
    const urlImg =
      "https://media4.giphy.com/media/jURBjTfNXLRKGzE042/giphy.gif?cid=42fdce0aa8mnyqbuz9c0ab1aksu9qf9vyogvabmdm6e2wx54&rid=giphy.gif";
    const wrapper = shallow(<GifGridItem urlImg={urlImg}/>);
    expect(wrapper).toMatchSnapshot()
  });
});
