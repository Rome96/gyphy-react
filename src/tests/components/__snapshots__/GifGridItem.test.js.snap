// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component <GifGridItem/> return component GifGridItem/> 1`] = `
<div
  className="card animate__animated animate__backInLeft"
>
  <h4 />
  <img
    src="https://media4.giphy.com/media/jURBjTfNXLRKGzE042/giphy.gif?cid=42fdce0aa8mnyqbuz9c0ab1aksu9qf9vyogvabmdm6e2wx54&rid=giphy.gif"
  />
</div>
`;
